flexmeasures_openweathermap/__init__.py

```diff
@@ -48,6 +49,10 @@
         description=f"Name of the weather station asset, defaults to '{DEFAULT_WEATHER_STATION_NAME}'",
         level="debug",
     ),
+    "MINIMAL_DISTANCE_DEGREE": dict(
+        descripion=f"Number of the minimum distance degree for weather station, defaults to '{DEFAULT_MINIMAL_DISTANCE_DEGREE}'",
```

Comment:
I think the english is a bit clunky, let's use this:

"Maximum distance (in degrees latitude & longitude) for weather stations from forecast location, defaults to {DEFAULT_MAXIMAL_DEGREE_LOCATION_DISTANCE}"